app/javascript/components/home/Home.jsx

```diff
@@ -1,8 +1,27 @@
 import React from 'react';
+import { Card, Stack } from 'react-bootstrap';
+import Button from 'react-bootstrap/Button';
+import Logo from '../shared_components/Logo';
 
 // This page is shown is the current_user does NOT have CreateRoom permission
 export default function Home() {
   return (
-    <>This page is for any user who doesnt have CreateRoom permission</>
+    <div className="mt-5">
+      <div className="text-center mb-4">
+        <Logo size="large" />
+      </div>
+      <Card className="col-md-8 mx-auto p-5 border-0 shadow-sm text-center">
+        <h1 className="mt-4"> Welcome to BigBlueButton. </h1>
+        <div className="mt-4 px-xxl-5">
+          <div className="text-start">
+            <h6> Please enter the URL of your meeting. </h6>
+            <Stack direction="horizontal">
+              <input className="form-control" id="joinUrl" />
```

Comment:
If it's possible, it's better to use the form template.
If it's not necessary we could use `react-bootstrap`, we're using native bootstrap when adding classes manually to native html elements.
It's not a big issue, but from best practices perspective it's good that we use `react-bootstrap` components like `Form.Control`.